lumpy/depthfilter.go

```diff
@@ -434,7 +441,7 @@ func interchromfilter(counts map[string]int, inters []inter) {
 					}
 				}
 				if c < len(buf) {
-					j = sort.Search(len(buf), func(z int) bool { return !distantLeftOutOfWindow(*buf[c], *buf[z]) })
+					j = max(j+1, sort.Search(len(buf), func(z int) bool { return !distantLeftOutOfWindow(*buf[c], *buf[z]) }))
```

Comment:
Why is this needed?

Reply:
This seems like it should be unnecessary, when paired with the change to excluding unmapped reads. Is that true? I'd worry leaving it in might mask other problems.

However, if left in place then I'd think we'd want to mirror this in line 420, 437 and 451 as well.

Reply:
The change within smoove to skip reads with unmapped mates should fix this, but I wanted to avoid any other condition that could result in non-termination.

Reply:
I think this can happen if you have a bunch of reads with the same tids and positions.